test/test_mps.py

```diff
@@ -8364,6 +8375,33 @@ def test_serialization_map_location(self):
 for t in [torch.double, torch.cdouble, torch.cfloat, torch.int8, torch.bfloat16]:
     del MPS_DTYPES[MPS_DTYPES.index(t)]
 
+abbrs_to_torch_dtype_dict = {value : key for (key, value) in dtype_abbrs.items()}
+class UnitTestSample:
+    def __init__(self, dtype, args, params, out):
+        requires_grad = (dtype.is_floating_point or dtype.is_complex)
+        self.args_ = [t.detach().to('mps').requires_grad_(requires_grad) for t in args]
+        self.params_ = params
+        self.out_ = out
+    def sample(self):
+        return self.args_ + self.params_
+    def expected(self):
+        return tuple(self.out_)
+
+CUDA_RESULT = dict()
+OP_UNIT_TEST = dict()
+with open("./test/cuda_results.yaml") as f:
```

Comment:
Denis fixed this path and is merged to master. This path seems to create some issues